src/definitions/modules/dropdown.js

```diff
@@ -297,6 +297,12 @@ $.fn.dropdown = function(parameters) {
             : module.get.query()
           ;
           module.verbose('Searching for query', query);
+          if(settings.fireOnInit === false && module.is.initialLoad()) {
+            module.verbose('Skipping callback on initial load', settings.onSearch);
+          }
+          else {
+            settings.onSearch.call(element, query);
```

Comment:
What about checking for a return value from that function to probably decline the search at all? We have this in other callbacks as well, so i suggest adding this to the minCharacters check one LOC later like this instead
```javascript
if(module.has.minCharacters(query) && settings.onSearch.call(element, query) !== false) {
```

Reply:
Seems like a good idea to remove that else statement and put it one line later